nncf/openvino/graph/model_transformer.py

```diff
@@ -354,9 +354,12 @@ def _set_const_value(node_with_const: ov.Node, const_port_id: int, const_value:
         if const_node is None:
             raise RuntimeError("Constant node was expected but could not find it.")
 
-        const_shape = const_node.get_data().shape
-        const_value = np.reshape(const_value, const_shape)
-        new_const_node = opset.constant(const_value, dtype=const_node.get_element_type())
+        const_shape = const_node.data.shape
+        const_dtype = const_node.data.dtype
+        const_value = np.reshape(const_value, const_shape).astype(const_dtype)
+
+        # TODO(andrey-churkin): Replace on opset13.constant() in a future release
+        new_const_node = ov.op.Constant(const_value, shared_memory=True)
```

Comment:
@alexsu52 @KodiaqQ The `shared_memory` parameter is not accessible in `opset13.constant()` for the our CI version of OV. It is available only on the master branch. Therefore, I am using `ov.op.Constant()` here. This will be updated in a future release.

Reply:
I'm just adding a note that using `ov.op.Constant` is not consistent with our current approach of using `opset` from Python API.